vinxictl/cli.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"runtime"
+	d "runtime/debug"
+	"strconv"
+	"time"
+
+	"gopkg.in/vinxi/vinxi.v0"
+)
+
+var (
+	aAddr         = flag.String("a", "", "bind address")
+	aPort         = flag.Int("p", 8080, "Port to listen")
+	aVers         = flag.Bool("v", false, "Show version")
+	aVersl        = flag.Bool("version", false, "Show version")
+	aHelp         = flag.Bool("h", false, "Show help")
+	aHelpl        = flag.Bool("help", false, "Show help")
+	aConfig       = flag.String("c", "", "Config file path")
+	aConfigl      = flag.String("config", "", "Config file path")
+	aForward      = flag.String("f", "", "Target server URL to forward traffic by default")
+	aReadTimeout  = flag.Int("http-read-timeout", 60, "HTTP read timeout in seconds")
```

Comment:
That's true. It has been ported from another code originally. The CLI implementation is a WIP now. It should be done from scratch.